client/angular/components/auth/form/auth_form.haml

```diff
@@ -4,10 +4,11 @@
     .auth-form__email-not-set.animated{ng-if: "!user.emailStatus"}
       %auth_provider_form{user: "user"}
       %auth_email_form{user: "user"}
+      .auth-form__privacy-notice.md-caption.lmo-hint-text{ng-if: "privacyUrl", translate: "auth_form.privacy_notice", translate-values: "{siteName: siteName, privacyUrl: privacyUrl}"}
     .auth-form__email-set.animated{ng-if: "user.emailStatus"}
-      %auth_identity_form.animated{ng-if: "pendingProviderIdentity", user: "user", identity: "pendingProviderIdentity"}
-      .auth-form__no-pending-identity.animated{ng-if: "!pendingProviderIdentity", ng-switch:"user.emailStatus"}
-        %auth_signin_form.animated{ng-switch-when:"active",   user: "user"}
-        %auth_signup_form.animated{ng-switch-when:"unused",   user: "user"}
-        %auth_inactive_form.animated{ng-switch-when:"inactive", user: "user"}
+      %auth_identity_form.animated{ng-if: "pendingProviderIdentity && !user.createAccount", user: "user", identity: "pendingProviderIdentity"}
+      .auth-form__no-pending-identity.animated{ng-if: "!pendingProviderIdentity || user.createAccount", ng-switch:"user.emailStatus"}
```

Comment:
I think the ng-switch wants to come off here.

The logic here is too complicated for my taste, I'd much rather do something like

```
.auth-form__no-pending-identity{ng-switch: "formToDisplay()"}
  %auth_signin_form{ng-switch-when: "signIn"}
  %auth_signup_form{ng-switch-when: "signUp"}
  %auth_inactive_form{ng-switch-when: "inactive"}
```
```
$scope.formToDisplay = ->
  if $scope.user.emailStatus == 'inactive'
    'inactive'
  else if $scope.user.name && ($scope.user.emailVerified || $scope.user.hasToken)
    'signIn'
  else
    'signUp'
```

I'd also love to move this logic to the serializer, since that's the only point (I think) of serializing out email_status, even the model if you're willing to go that far.
```
# pending::baseserializer.rb
if user.deactivated_at
  'inactive'
elsif (user.email_verified || has_token) && user.name
  'signIn'
else
  'signUp'
end
```
```
# after serializing that:
ng-switch-when: "user.authForm"
```